### ISSUE TITLE
Rubick bug

### ISSUE THREAD

**Issue by qoval1** [jeffhill/Dota2#545]
### Ability name

Telekinesis

### Description

Rubick's telekinesis interaction bug with Black hole, Duel, Chronosphere. Chainstunning into those spells are not avaliable
![2022-06-03_14-10-56 (1)](https://user-images.githubusercontent.com/106869198/171996945-5cddc713-131a-4729-8eff-3a650dead56c.gif)

### Example Match ID (and possibly Timestamp)

don't remember match id, sorry

**Comment by MPatersonSoftware** [comment]
What's the alternative for this? The desired outcome? Specifically for BH and Chrono. 

This seems to be forced movement vs forced movement and telekinesis is just lower in the pecking order, the alternative would seem to be telekinesis goes off but is immediately overridden by the 'stronger' forced movement of the ults... at least this way it doesn't go on cooldown for nothing? 

Upvoting because Duel seems to be a bug imo though.

**Comment by qoval1** [comment]
if it's not wtf mode, telekenesis going to cd without any effect, so it's imposible to chain with telekenesis

**Comment by johnsonloh6185** [comment]
> if it's not wtf mode, telekenesis going to cd without any effect, so it's imposible to chain with telekenesis

yes it will go to cd for sure because you succesful casting your spell but it dosent meant your spell will affected the unit you selected, same goes to if you using Tiny's Tree Volley when nearby dont have any tree, the skill you cast is succesful but because nearby dont have any tree so it goes to CD as used skill

**Comment by johnsonloh6185** [comment]
> What's the alternative for this? The desired outcome? Specifically for BH and Chrono.
> 
> This seems to be forced movement vs forced movement and telekinesis is just lower in the pecking order, the alternative would seem to be telekinesis goes off but is immediately overridden by the 'stronger' forced movement of the ults... at least this way it doesn't go on cooldown for nothing?
> 
> Upvoting because Duel seems to be a bug imo though.

Telekenesis listed as a spell that cant move Duel unit

**Comment by Dopwop** [comment]
see #190
In my opinion the fact that Telekinesis consumes mana and goes on cooldown while doing nothing is still a bug. Either it should still stun (without any forced movement) or the ability should not be castable (like on BKB).

**Comment by DarkEyeDragon** [comment]
> see #190 In my opinion the fact that Telekinesis consumes mana and goes on cooldown while doing nothing is still a bug. Either it should still stun (without any forced movement) or the ability should not be castable (like on BKB).

This seems like correct behavior. The hero is not magic immune and is not being moved (thus not stunned). It is the lift and the "slam" into the ground that stuns the target, but if the target is literally stopped in time, or being consumed into a black hole i don't think you can really apply those forces. Thats just my opinion though.

**Comment by qoval1** [comment]
> > What's the alternative for this? The desired outcome? Specifically for BH and Chrono.
> > This seems to be forced movement vs forced movement and telekinesis is just lower in the pecking order, the alternative would seem to be telekinesis goes off but is immediately overridden by the 'stronger' forced movement of the ults... at least this way it doesn't go on cooldown for nothing?
> > Upvoting because Duel seems to be a bug imo though.
> 
> Telekenesis listed as a spell that cant move Duel unit

Same mechanics with lasso, but it's works and u can stun with telekenesis
